web/docs/tql2/operators/assert_throughput.md

```diff
@@ -0,0 +1,26 @@
+# assert_throughput
+
+Emits a warning if the pipeline does not have the expected throughput
+
+```tql
+assert min_events:int, within=duration, [retries=int]
```

Comment:
Did you consider making the minimum named as well?

Reply:
The arguments are aligned with the `throttle` operator, which I think is better.